README.md

```diff
@@ -1,12 +1,18 @@
 # Sushiksha-website 
 
+<p class="text-center mb-3" align="center">
```

Comment:
At the top add Sushiksha Logo
Make it center aligned

Reply:
Where can i get the logo of sushiksha ?

Reply:
https://vkssfalumni.com/wp-content/uploads/2016/11/sushikshalogo-300x300.png